docs/source/android-section.md

```diff
@@ -0,0 +1,57 @@
+# Android
+
+## Quick Start & Integration
+[Getting started with ExecuTorch on Android](using-executorch-android)
+
+## Available Backends
+
+### CPU Backends
+- [XNNPACK Backend](backends-xnnpack) - Optimized CPU inference
```

Comment:
When I click on this https://docs-preview.pytorch.org/pytorch/executorch/14720/backends-xnnpack.html it goes to iOS section.